rule/comment-spacings.go

```diff
@@ -0,0 +1,74 @@
+package rule
+
+import (
+	"go/ast"
+
+	"github.com/mgechev/revive/lint"
+)
+
+// CommentSpacings Rule check the whether there is a space between
+// the comment symbol( // ) and the start of the comment text
+type CommentSpacingsRule struct{}
+
+func (*CommentSpacingsRule) Apply(file *lint.File, _ lint.Arguments) []lint.Failure {
+	var failures []lint.Failure
+	fileAST := file.AST
+	onFailure := func(failure lint.Failure) {
+		failures = append(failures, failure)
+	}
+	whiteList := map[string]bool{
+		"//go:generate":              true,
```

Comment:
I'm removing the whiteList and just checking if a comment text has a Prefix `go:` or`revive:`.

Reply:
Why I am not thinking of refactoring to a list anymore is because, let say we store the comments that have the above given prefix, the string matching from the `allowList` will still have the same complexity as `strings.HasPrefix`, in fact more.